src/main/java/com/mraof/minestuck/block/StoneTabletBlock.java

```diff
@@ -0,0 +1,154 @@
+package com.mraof.minestuck.block;
+
+import com.mraof.minestuck.client.gui.MSScreenFactories;
+import com.mraof.minestuck.tileentity.ItemStackTileEntity;
+import com.mraof.minestuck.util.CustomVoxelShape;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.Blocks;
+import net.minecraft.block.material.PushReaction;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.state.BooleanProperty;
+import net.minecraft.state.StateContainer;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.ActionResultType;
+import net.minecraft.util.Direction;
+import net.minecraft.util.Hand;
+import net.minecraft.util.IStringSerializable;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.BlockRayTraceResult;
+import net.minecraft.util.math.RayTraceResult;
+import net.minecraft.world.IBlockReader;
+import net.minecraft.world.IWorld;
+import net.minecraft.world.World;
+import net.minecraft.world.storage.loot.LootContext;
+import net.minecraft.world.storage.loot.LootParameters;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class StoneTabletBlock extends DecorBlock //stone slab is the same as stone tablet, both are used in different circumstances
+{
+	public static final BooleanProperty CARVED = MSProperties.CARVED;
+	
+	public StoneTabletBlock(Properties properties, CustomVoxelShape shape)
```

Comment:
Since this block would probably not be used with any other shapes, I think it's fine to remove the shape parameter and specify the shape directly here instead of in MSBlocks. Do you intentionally want to specify the shape in MSBlocks, or is this parameter something that just came along from elsewhere?
(Note: this is not important)